Tests/test_imagemath_lambda_eval.py

```diff
@@ -19,7 +23,7 @@ def pixel(im: Image.Image | int) -> str | int:
 A2 = A.resize((2, 2))
 B2 = B.resize((2, 2))
 
-images = {"A": A, "B": B, "F": F, "I": I}
+images: dict[str, Any] = {"A": A, "B": B, "F": F, "I": I}
```

Comment:
Does it work?
```suggestion
images: dict[str, Image.Image] = {"A": A, "B": B, "F": F, "I": I}
```

Reply:
Not exactly, no.
```
Tests/test_imagemath_lambda_eval.py:37: note: "Dict" is invariant -- see https://mypy.readthedocs.io/en/stable/common_issues.html#variance
Tests/test_imagemath_lambda_eval.py:37: note: Consider using "Mapping" instead, which is covariant in the value type
Tests/test_imagemath_lambda_eval.py:43: error: Argument 2 to "lambda_eval" has incompatible type "**dict[str, Image]"; expected "dict[str, Any]"  [arg-type]
```
This would be because `**kw` is `Any`.
https://github.com/python-pillow/Pillow/blob/47fc36a323892040b1945afd22d036a15fb99abd/src/PIL/ImageMath.py#L239-L243

Reply:
It looks strange. Isn't `Image` `Any`?

Reply:
I think the key part of mypy's response is

> "Dict" is invariant -- see https://mypy.readthedocs.io/en/stable/common_issues.html#variance

`Image` may sit within `Any`, but it is not `Any`. I don't think there's a better option, short of removing `images` altogether and just using the image variables individually.